src/containers/AppContainer.tsx

```diff
@@ -0,0 +1,43 @@
+import { Navigate, Route, Routes } from 'react-router-dom';
+
+import { ProtectedRoute } from '@components/common';
+import { ResetPasswordContainer } from 'containers';
+
+import { settingsInitialState, SettingsContext } from 'contexts';
+import { useGetLoggedInPersonnel, useToken } from 'hooks';
+
+export const AppContainer = () => {
+    const { token } = useToken();
+    const { data: personnel, isLoading } = useGetLoggedInPersonnel({
+        enabled: !!token
+    });
+
+    return token && isLoading ? (
+        <></>
+    ) : (
+        <SettingsContext.Provider
+            value={{
+                ...settingsInitialState,
```

Comment:
why are we copying code from dashboard? we don't need all of this info here. Pls revert

Reply:
removed.